packages/usa-nav/src/styles/_usa-nav.scss

```
@@ -473,3 +473,17 @@ $expand-less-icon: map-merge(
 .usa-js-mobile-nav--active {
   overflow: hidden;
 }
+
+// Safari-only fix that forces a vertical scrollbar when mobile header is opened.
+// Fix is only needed in the 15px immediately preceding $theme-header-max-width.
+// Note: 15px is the current width of the Safari scrollbar.
+// More details in https://github.com/uswds/uswds/pull/5443
+$safari-header-bug-min-width: calc(
+  units($theme-header-max-width) - px-to-rem(15px)
+);
+
+@media (min-width: $safari-header-bug-min-width) {
+  .usa-js-mobile-nav--active.is-safari {
+    overflow: scroll;
```

I was curious what would happen if we try setting `scrollbar-width: initial;` on this line. Currently unable to recreate the bug so passing the recommendation for now! I was hoping this would keep the inital scrollbar width as well as the fixed scrolling

```suggestion
    scrollbar-width: initial;
```

---

This unfortunately did not work. Without `overflow: scroll`, the scrollbar disappeared and `scrollbar-width` did not force the scrollbar width.